docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -31,6 +31,13 @@ following steps:
    API server, each with its own identity, as well as an additional
    kubeconfig file for administration named `admin.conf`.
 
+1. If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig` enabled,
+   writes the kubelet init configuration into `/var/lib/kubelet/config/init/kubelet` file. 
```

Comment:
>writes the kubelet init configuration into `/var/lib/kubelet/config/init/kubelet` file.

it writes the kubelet init configuration into the `/var/lib/kubelet/config/init/kubelet` file.